modules/ducktests/tests/ignitetest/tests/discovery_test.py

```diff
@@ -149,18 +179,134 @@ def _perform_node_fail_scenario(self, test_config):
 
             start_load_app(self.test_context, ignite_config=load_config, params=params, modules=modules)
 
-        data = simulate_nodes_failure(servers, failed_nodes, survived_node)
+        for node in failed_nodes:
+            self.logger.info(
+                "Simulating failure of node '%s' (order %d) on '%s'" % (node_id(node), order(node), node.name))
+
+        data = self._simulate_nodes_failure(servers, node_fail_task(ignite_config, test_config), failed_nodes,
+                                            survived_node)
 
         data['Ignite cluster start time (s)'] = start_servers_sec
 
         return data
 
+    def _simulate_nodes_failure(self, servers, kill_node_task, failed_nodes, survived_node):
+        """
+        Perform node failure scenario
+        """
+        ids_to_wait = [node_id(n) for n in failed_nodes]
+
+        _, first_terminated = servers.exec_on_nodes_async(failed_nodes, kill_node_task)
+
+        for node in failed_nodes:
+            self.logger.debug(
+                "Netfilter activated on '%s': %s" % (node.name, dump_netfilter_settings(node)))
+
+        # Keeps dates of logged node failures.
+        logged_timestamps = []
+        data = {}
+
+        for failed_id in ids_to_wait:
+            servers.await_event_on_node(failed_pattern(failed_id), survived_node, 15, from_the_beginning=True,
+                                        backoff_sec=0.3)
+
+            _, stdout, _ = survived_node.account.ssh_client.exec_command(
```

Comment:
We can't until we stabilize count of IP addresses and merge IGNITE-13465.

Reply:
The goal is to see timings and seeing them find ways to improve.
If we have bad timings now on other nodes let's see this.

Reply:
My goal for this ticket was to see timings in the logs and have ability to compare with another ignite versions. Timings appear strange. But test works as it can with current bug in discovery and the disadvantages with multiple IP addresses. Test show timings but was not supposed to do assertions. Is it another ticket?

Reply:
Another ticket, not a problem. Just mentioned that we need to have more info.